code/dynamic_programming/src/fibonacci/fibonacci.cpp

```diff
@@ -0,0 +1,47 @@
+/* Part of Cosmos by OpenGenus Foundation */
+
+#include <stdlib.h>
+#include <sstream>
+#include <iostream>
+#include <vector>
+
+int get_fib(int num);
+
+int main()
+{
+    using namespace std;
+  
+    int number;
+
+    cout << "Enter the value of n: ";
+    cin >> number;
+  
+    if(number > 46 || number < 0)
+    {
+        cout << "Must enter an integer between 0 and 46\n";
+        exit(0);
+    }
+    
+    int n_fib = get_fib(number);
+
+    cout << "Fibonacci number is: " << n_fib << endl;
+
+    return 0;
+}
+
+int get_fib(int num)
+{
+    using namespace std;
```

Comment:
Why to keep namespace in individual functions?
Instead I would suggest you to keep it global.